sc-tools/sc-config-utils/sc_memory_config.cpp

```diff
@@ -60,28 +53,28 @@ ScMemoryConfig::ScMemoryConfig(ScConfig const & config, ScParams const & params,
     std::string value = group[key];
     value = !value.empty() && value[0] == '\"' ? value.substr(1, value.length() - 2) : value;
 
-    m_params.insert({key, value});
+    m_params.Insert({key, value});
   }
 }
 
-sc_char const * ScMemoryConfig::GetStringByKey(std::string const & key, sc_char const defaultValue[])
+sc_char const * ScMemoryConfig::GetStringByKey(std::string const & key, sc_char const * defaultValue)
 {
-  return m_params.count(key) ? m_params.at(key).c_str() : (sc_char const *)defaultValue;
+  return m_params.Has(key) ? m_params.Get<std::string>(key).c_str() : defaultValue;
```

Comment:
I don't really get why codestyle fixes were made in this PR and not in a separate one - this decision makes it very difficult to review the PR since the attention is scattered and you have to spend time to understand whether the change is codestyle-related or functional

Reply:
I make separated commits for such changes